apiserver/plane/app/views/issue/link.py

```diff
@@ -44,6 +45,9 @@ def create(self, request, slug, project_id, issue_id):
         serializer = IssueLinkSerializer(data=request.data)
         if serializer.is_valid():
             serializer.save(project_id=project_id, issue_id=issue_id)
+            crawl_work_item_link_title(
```

Comment:
@sangeethailango @sriramveeraghanta This can increase the response time of the request if the link takes too long to load while crawling.